rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -522,6 +522,7 @@ def _make_stored_sample_dict_from_row(bss_row, has_order=True):
                         where bss2.biobank_order_identifier = boi.`value`) as tests_stored
              from biobank_order bo left outer join biobank_order_identifier boi on bo.biobank_order_id = boi.biobank_order_id
              where boi.`system` = 'https://www.pmi-ops.org' and bo.participant_id = :pid
```

Comment:
I think this is tangential to your changes, but might be worth discussion. Will this need to include orders that are coming from Quest/CE?